### ISSUE TITLE
HMW design in usernames?

### ISSUE THREAD

**Issue by fannyBkane** [emfoundation/ce100-app#1494]

**Comment by asimonw** [comment]
The plan could be as follows:
* Iteration 1: only @ people, not organisation. The format of the username could be `first-name.last-name` (all lowercase), autogenerated on save and can only be edited by admins.
* Iteration 2: In the search results (as the user is typing) we also display organisation and whether the person is primary contact.
* Iteration 3: In order to show which organisation an @'ed person belongs to, we show the org (and possibly other info such as job title and whether they are a primary contact) on hover (what happens on mobile?)

@Kimberley-Heath Does this correspond to what you understood from our convo?

**Comment by fannyBkane** [comment]
@asimonw To clarify you wouldn't be able to @' an organisation i.e. **@orange** ?

**Comment by asimonw** [comment]
@fannyBkane Indeed, at least not as part of the first iteration. There are too many questions around this, so it seems better to make sure the "simpler" case works well first. wdyt?

**Comment by fannyBkane** [comment]
@asimonw Yes it makes sense 👍
